app/src/main/java/ch/epfl/sdp/contamination/fragment/UserInfectionFragment.java

```diff
@@ -235,25 +229,15 @@ private void executeHealthStatusChange() {
     }
 
     private void sendRecoveryToFirebase() {
-        // TODO: @Lucie I think that doesn't upload anything to Firestore
+        // TODO: I think that doesn't upload anything to Firestore
+        // --->> Are you sure? From what I read on the doc,
+        //                      update method from DocumentReference updates on Firebase
```

Comment:
From my tests of the app the infection status seems to be preserved accorss app launchs